CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Added
 * Add button to Bugzilla on public and private comments
 * `DEFER` is now a possible affect resolution (`OSIDB-3286`, `OSIDB-3288`)
+* Allow using default ordering in flaw list page (`OSIDB-3187`)
```

Comment:
This should be inside the `Unreleased section` update your branch with main, #392 added the section